src/Umbraco.Web.UI.Client/src/views/components/umb-content-grid.html

```diff
@@ -9,7 +9,7 @@
       <i ng-if="item.selected" class="icon-check umb-content-grid__checkmark"></i>
 
       <div class="umb-content-grid__icon-container">
-         <i class="{{ item.icon }} umb-content-grid__icon" ng-class="{'-light': !item.published}"></i>
+          <i class="umb-content-grid__icon {{ item.icon }}" ng-class="{'-light': !item.published}"></i>
       </div>
```

Comment:
Apparently it seems to matter the order of these classes, but since ```__icon``` is first in other views, I have also moved it as the first class here.